main.qml

```diff
@@ -152,7 +153,9 @@ Window {
                 textMargin: 12;
                 width:1404;
                 textFormat: mode == 0 ? TextEdit.RichText : TextEdit.PlainText;
-                font.family: mode == 0 ? "Noto Sans" : "Noto Mono";
+                //font.family: mode == 0 ? "Noto Sans" : "Noto Mono";
+		font.family: mode == 0 ? "Noto Serif" : "Hack";
```

Comment:
I'm curious why you want to change the font face/family as well as the size?

Reply:
My bad. This would comprise only the size change.

Maybe it‘s just me but the edit mode font size is just way too large?
E.g. compare to the default font size of xochitl ui.

Reply:
Great - would you mind updating your PR to remove this font family change and then I will merge it?